SR2026/2025-09-09-competition-announcement.md

```diff
@@ -0,0 +1,46 @@
+---
+to: Potential team supervisors and SR2025 teams
+subject: Announcing Student Robotics 2026!
+preview: |
+  Sign up to Student Robotics 2026 and get started with our pre-kickstart activities!
+---
+
+Hi,
+
+We're pleased to [announce][announcement] that registration for Student Robotics
+2026 is now open. The [programme][programme-structure] will run from our Kickstart event in November to the in-person competition over two days around Easter 2026.
+
+If you would like to compete, please complete the [signup form][signup-form]
+with the details of your team. Entries must be made by a responsible adult on
+behalf of a team -- we call this role a [team supervisor][team-supervisor]. All
+future emails about SR2026 will be sent to the details you provide via the form.
+
+This year we're hosting [Kickstart][kickstart] at the University of Southampton,
+on Saturday 8th November. We will also be live-streaming the event for those
+unable to attend in person. We expect to confirm places towards the end
+of September. Places are limited, so sign up soon to avoid disappointment.
+
+As usual we will loan competing teams a [kit][kit] of parts to help build their
+robots. Teams can collect their kit from the Kickstart event, so they can get started straight away. If you're unable to attend in person, we aim to ship your kit to you prior to the event. We'll confirm further details about this once you've registered for the year.
+
+We have also created some [pre-Kickstart activities][pre-kickstart-activities]
+which enable anyone to start learning about robotics and our simulator.
```

Comment:
The pre-Kickstart activities will be coming soon, that page is out of date. Just direct them to follow the instructions for installing the simulator for now.

Reply:
We should still mention that they are coming and that they can get the simulator setup, pending https://github.com/srobo/docs/pull/684